### PR TITLE
Add  sUDT_ERC20_Proxy contract test case for Testnet v1

### PR DESCRIPTION
**Pull request by Flouse**
##  sUDT-ERC20 Proxy Contracts
```js
{
  symbol: 'GWK',
  name: 'GodwokenToken',
  decimals: 18,
  addr: '0x2275Afe815dE66BeAbE7A2C03005537AB843afB2',
  totalSupply: BigNumber { value: "1820" }
}
```

### CODE REVIEW

File: .github/workflows/contract-test.yml [hunk 1]
```diff
@@ -66,3 +66,4 @@ jobs:
       env:
         PRIVATE_KEY: ${{ secrets.GODWOKEN_TEST_PRIVATE_KEY }}
         PRIVATE_KEY2: ${{ secrets.GODWOKEN_TEST_PRIVATE_KEY2 }}
+        PRIVATE_KEY3: ${{ secrets.GW_TESTNET_V1_TEST_PK }}
```
Comment (ShookLyngs) [inline .github/workflows/contract-test.yml:70]: Should we add the new secret in job `check-secrets`?

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: .github/workflows/contract-test.yml (hunk 1 above)
Comment (Flouse) [reply .github/workflows/contract-test.yml:70]: 1b3727985540ad7905fa7d042f66d4b47fef07ca

File: .github/workflows/contract-test.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@ jobs:
         env:
           KEY1: '${{ secrets.GODWOKEN_TEST_PRIVATE_KEY }}'
           KEY2: '${{ secrets.GODWOKEN_TEST_PRIVATE_KEY2 }}'
+          KEY3: '${{ secrets.GW_TESTNET_V1_TEST_PK }}'
         if: ${{ env.KEY1 != '' && env.KEY2 != '' }}
```
Comment (ShookLyngs) [inline .github/workflows/contract-test.yml]: Should add the `KEY3` condition check:
```yml
if: ${{ env.KEY1 != '' && env.KEY2 != '' && env.KEY3 != '' }}
```

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: contracts/hardhat.config.js [hunk 3]
```diff
@@ -4,10 +4,13 @@ require("@nomiclabs/hardhat-web3");
...
 // eth_address: 0x966b30e576a4d6731996748b48dd67c94ef29067
-const PRIVATE_KEY = process.env.PRIVATE_KEY ?? "1390c30e5d5867ee7246619173b5922d3b04009cab9e9d91e14506231281a997";
+const TEST_PK1 = process.env.PRIVATE_KEY ?? "1390c30e5d5867ee7246619173b5922d3b04009cab9e9d91e14506231281a997";
 
 // eth_address: 0x4fef21f1d42e0d23d72100aefe84d555781c31bb
-const PRIVATE_KEY2 = process.env.PRIVATE_KEY2 ?? "2dc6374a2238e414e51874f514b0fa871f8ce0eb1e7ecaa0aed229312ffc91b0";
+const TEST_PK2 = process.env.PRIVATE_KEY2 ?? "2dc6374a2238e414e51874f514b0fa871f8ce0eb1e7ecaa0aed229312ffc91b0";
+
+// The issuer of GWK, eth_address: 0x8291507afda0bba820efb6dfa339f09c9465215c
+const TEST_PK3 = process.env.PRIVATE_KEY3;
```
Comment (ShookLyngs) [inline contracts/hardhat.config.js]: Need a default value for `TEST_PK3`, so `gw_testnet_v1` tests can be run in the local environment more conveniently.

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: contracts/hardhat.config.js (hunk 3 above)
Comment (Flouse) [reply contracts/hardhat.config.js]: [chore: add a default TEST_PK containing some GWK](https://github.com/nervosnetwork/godwoken-tests/pull/134/commits/0be628a6d6d17076dfb6ba80da801713f6e55c36)

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**